docs/aws/audit/route53monitoring/rules/domain_auto_renew.mdx

```diff
@@ -23,6 +23,205 @@ CBP
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all the hosted zones: Once the AWS CLI is set up, you can list all the hosted zones in your AWS account by running the following command:
+   ```
+   aws route53 list-hosted-zones
+   ```
+   This command will return a list of all the hosted zones along with their details.
+
+3. Get the details of each hosted zone: For each hosted zone, you can get more details including the domain name, record set count, and more importantly, whether the auto-renew is enabled or not. Use the following command to get the details of a specific hosted zone:
+   ```
+   aws route53 get-hosted-zone --id <HostedZoneId>
+   ```
+   Replace `<HostedZoneId>` with the ID of the hosted zone you want to check. This command will return the details of the specified hosted zone.
+
+4. Check the Auto Renew status: In the output of the previous command, look for the `Config` attribute. If the `AutoRenew` attribute under `Config` is set to `true`, then auto-renew is enabled for that hosted zone. If it's set to `false`, then auto-renew is not enabled.
+
+Please note that AWS Route 53 does not directly support the auto-renew feature. The auto-renew feature is generally associated with domain registration services. AWS Route 53 does support domain registration, and when you register a domain, auto-renew is enabled by default. However, this setting can be changed in the Route 53 domain registration settings, not in the hosted zone settings.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'
+)
+```
+
+3. Use the Route53 client from the session to list all the hosted zones:
+
+```python
+route53_client = session.client('route53')
+
+response = route53_client.list_hosted_zones()
+
+for hosted_zone in response['HostedZones']:
+    print("Hosted Zone: ", hosted_zone['Name'])
+```
+
+4. For each hosted zone, list all the domain names and check if auto renew is enabled:
+
+```python
+for hosted_zone in response['HostedZones']:
+    hosted_zone_id = hosted_zone['Id']
+    domain_list = route53_client.list_domains(HostedZoneId=hosted_zone_id)
+    
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>